usecases/auth/authorization/rbac/authorizer.go

```diff
@@ -77,3 +90,81 @@ func (m *manager) Authorize(principal *models.Principal, verb string, resources
 func (m *manager) AuthorizeSilent(principal *models.Principal, verb string, resources ...string) error {
 	return m.authorize(principal, verb, true, resources...)
 }
+
+func (m *manager) FilterAuthorizedResources(principal *models.Principal, verb string, resources ...string) ([]string, error) {
+	if principal == nil {
+		return nil, errors.NewUnauthenticated()
+	}
+
+	if len(resources) == 0 {
+		return nil, fmt.Errorf("at least 1 resource is required")
+	}
+
+	logger := m.logger.WithFields(logrus.Fields{
+		"action":         "authorize",
+		"user":           principal.Username,
+		"component":      authorization.ComponentName,
+		"request_action": verb,
+	})
+	if len(principal.Groups) > 0 {
+		logger = logger.WithField("groups", principal.Groups)
+	}
+
+	var permResults []logrus.Fields
+	var allowedResources []string
+
+	// First try with wildcard
+	parts := strings.Split(resources[0], "/")
+	parts[len(parts)-1] = "*"
+	wildcardPath := strings.Join(parts, "/")
+
+	allowed, err := m.checkPermissions(principal, wildcardPath, verb)
+	if err != nil {
+		logger.WithError(err).WithField("resource", wildcardPath).Error("failed to enforce policy")
+		return nil, err
+	}
+
+	if allowed {
+		// If wildcard is allowed, log permissions for all requested resources
```

Comment:
shouldn't we log the wildcard path for this case?

Reply:
we log at [L141](https://github.com/weaviate/weaviate/pull/7196/files/86e27bead454f4c183dfafcb38c0931d4914b47e#diff-aa4d0b0c5c2e4f6f0f0b165b03b9ee9a61cf1d16420a2147f5a49a8ff4f43ffaR141)